applications/compressible_navier_stokes/turbulent_channel/application.h

```diff
@@ -409,6 +409,25 @@ class Application : public ApplicationBase<dim, Number>
     pp_data.output_data.degree             = this->param.degree;
     pp_data.output_data.write_higher_order = false;
 
+    // write data to hdf5
+    pp_data.pointwise_output_data.write_output = true;
```

Comment:
in the end, this should be set to `false` (and activated by the user) in my opinion

Reply:
resolved in [a674dae](https://github.com/exadg/exadg/pull/255/commits/a674dae71e792a298cc4cef22fc65d0f5b3598f7)